harvard-university-of-leeds.csl

```diff
@@ -32,7 +36,7 @@
   </macro>
   <macro name="author">
     <names variable="author">
-      <name and="text" delimiter-precedes-last="never" initialize-with="." name-as-sort-order="all"/>
+      <name and="text" delimiter-precedes-last="never" et-al-min="0" initialize-with="." name-as-sort-order="all"/>
```

Comment:
it first falls back to what's in the bibliography, then to what's in style, then it doesn't do et al ever. But it appears to be more robust to just set it to a high number, which is why I'm recommending that.